docs/sources/writing-guide/front-matter/index.md

```diff
@@ -82,42 +92,73 @@ For `labels.products`, the supported values and the resulting published labels a
 - `enterprise`: "Enterprise"
 - `oss`: "Open source"
 
-Labels can be inherited through cascading front matter.
-Each project has a set of default labels that are defined in the root `_index.md` file of the project.
+Labels are inherited through cascading front matter.
+All labels, including project defaults, are set in the project's root index file.
+This is almost always the `docs/sources/_index.md` file in the project repository.
 
-For versioned projects, the `_index.md` file resides in the `website` repository.
-For unversioned projects, the `_index.md` file resides in the project’s repository.
+If you add a new page, consider whether the default labels are appropriate.
+If the default labels are incorrect for a page or directory of pages, update the cascading front matter in `docs/sources/_index.md`.
 
-If the default labels are incorrect for a page or directory of pages, update the labels.
-Also, if you are adding a new page, consider whether the default labels are appropriate.
-For each page, include a label in the `labels.products` sequence for every product that the page relates to.
+#### Set cascading front matter
 
-For example, if a **single page** describes a feature available in Grafana Cloud and Grafana Enterprise, the source file front matter should include the following:
+Set cascading front matter by adding or updating the `cascade` mapping in the `docs/sources/_index.md` front matter:
```

Comment:
From here I suggest deleting everything from line 105-122. And then simply showing the example that starts with the intro at line 123. It's a much more efficient way to communicate to non-technical person what to do and exactly how.

```suggestion
Set cascading front matter by adding or updating the `cascade` mapping in the `docs/sources/_index.md` front matter.
```